### ISSUE TITLE
Encountered `failed: Connection refused`

### ISSUE THREAD

**Issue by yxia-2022** [castacks/tartanair_tools#51]
Hi TartanAir Maintainer,

Firstly, thank you so much for providing such a great dataset! 

To download the dataset, I've encountered a connection problem. An example of the failure message is
```
--2024-04-28 21:34:36--  https://airlab-share.andrew.cmu.edu:8081/tartanair/westerndesert/Hard/image_right.zip                                                                          
Resolving airlab-share.andrew.cmu.edu (airlab-share.andrew.cmu.edu)... 128.237.74.70
Connecting to airlab-share.andrew.cmu.edu (airlab-share.andrew.cmu.edu)|128.237.74.70|:8081... failed: Connection refused.
```

Could anyone help and check the server status?

Best,
Yukun

**Comment by yxia-2022** [comment]
Nope, although I can connect to `128.237.74.70`, port `8081` is not accessible. I tried to look for some backup resources. But, no luck 🤷

**Comment by ZekaiYinPKU** [comment]
I emailed the maintainer, they replied that they are fixing it. I think it will back online soon:)

**Comment by chanyoungl** [comment]
Has anyone downloaded it?
I still have connection refused.

**Comment by ZekaiYinPKU** [comment]
Now on my side it turns to connection timed out

**Comment by leofernandeso** [comment]
> Now on my side it turns to connection timed out

Same here

**Comment by SebastianHeid** [comment]
I also tried to download the dataset and I got the error:
`ERROR 503: Service Unavailable`
The directories were created correctly but with empty zip files.

**Comment by zhengqili** [comment]
It seems that the server is still down

**Comment by smash0190** [comment]
We are working on getting it back up. This time will be cloud hosted

**Comment by leofernandeso** [comment]
> We are working on getting it back up. This time will be cloud hosted

Hi, were there any updates here? It would be very cool if we can use this dataset again.

Thanks!
